content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,132 @@
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/uefi.png" src="/img/blogs/newsletter-103/uefi.png" caption="Running ReactOS on a Steam Deck" >}}
+{{< figure link="/img/blogs/newsletter-103/htc_hd2.png" src="/img/blogs/newsletter-103/htc_hd2.png" caption="Running FreeLoader on a HTC HD2" >}}
+{{< figure link="/img/blogs/newsletter-103/lumia950xl.png" src="/img/blogs/newsletter-103/lumia950xl.png" caption="Running FreeLoader on a Lumia 950XL" >}}
+{{< figure link="/img/blogs/newsletter-103/bootmgfw_freeldr.png" src="/img/blogs/newsletter-103/bootmgfw_freeldr.png" caption="ReactOS FreeLoader boot entry shown in Windows Boot Manager in a Framework laptop" >}}
+{{< figure link="/img/blogs/newsletter-103/apple.png" src="/img/blogs/newsletter-103/apple.png" caption="Running FreeLoader on a Apple iPhone 5c" >}}
+{{< /gallery >}}
+
+It's worth noting that FreeLoader runs natively on the aforementioned phones, without emulators of any kind.
+
+**WARNING:** UEFI support in ReactOS is experimental, under heavy development and is not yet ready for the general use. We currently cannot give an estimate as to when it will be done, so please stay tuned!
+
+## Shell improvements
+Doug Lyons, Katayama Hirofumi MZ and Carl Bialorucki (cbialorucki) are extensively improving the Shell component of ReactOS to provide a better user experience with the operating system.
+Notably Doug Lyons improved the toolbar icon display in MS Office 2000 and other programs, by fixing some bugs in the internal functions (`SetDIBitsToDevice` and `NtGdiSetDIBitsToDeviceInternal`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/before.png" src="/img/blogs/newsletter-103/before.png" caption="BEFORE" >}}
+{{< figure link="/img/blogs/newsletter-103/after.png" src="/img/blogs/newsletter-103/after.png" caption="AFTER" >}}
+{{< /gallery >}}
+
+Katayama improved the ZIP archive file support to allow zipping or unzipping files and folders that have UTF-8 characters in them.
+This will greatly help with the interoperability between systems with different language settings.
+In addition to that, we can choose the code-page of ZIP compression/extraction by using the following registry key (`HKCU\Software\ReactOS:ZipCodePage / UnZipCodePage`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/utf8.png" src="/img/blogs/newsletter-103/utf8.png" caption="Extracting from a ZIP file that has Japanese characters on a Chinese locale system">}}
+{{< /gallery >}}
+
+Katayama added an "Internet Browser" icon in the desktop workspace, which will open the default web browser that is installed onto the computer.
+In the future this icon can be hidden through Shell settings.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/internet.png" src="/img/blogs/newsletter-103/internet.png" >}}
+{{< /gallery >}}
+
+Carl added support for large taskbar icons in the Explorer component of the shell. This can help with people with visual impairments.
```

Comment:
```suggestion
Carl added support for large taskbar icons in the Explorer component of the shell.
This can help with people with visual impairments.
```